api/registrations/serializers.py

```diff
@@ -299,22 +299,14 @@ class RegistrationSerializer(NodeSerializer):
         read_only=True,
     )
 
-    links = LinksField({'self': 'get_registration_url', 'html': 'get_absolute_html_url'})
-
-    def get_registration_url(self, obj):
-        return absolute_reverse(
-            'registrations:registration-detail', kwargs={
-                'node_id': obj._id,
-                'version': self.context['request'].parser_context['kwargs']['version'],
-            },
-        )
+    links = LinksField({'html': 'get_absolute_html_url'})
```

Comment:
Logic elsewhere to add view_only query param to `self` link, so using that instead of describing self link here.